web/react/components/channel_header.jsx

```diff
@@ -95,6 +95,7 @@ module.exports = React.createClass({
     _onChange: function() {
         var newState = getStateFromStores();
         if (!utils.areStatesEqual(newState, this.state)) {
+            newState.last_edit_time = new Date();
```

Comment:
We might want to use Date.now instead of new Date objects for keeping track of the last edit time since it's faster.